package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "mes-microservices-starter-template",
+  "version": "0.1.0",
+  "author": "DVSA",
+  "license": "MIT",
+  "description": "A starter template for lambda microservices in the MES project",
+  "homepage": "https://github.com/dvsa/mes-microservices-starter-template#README.md",
+  "repository": {
```

Comment:
Need the name,version, description, homepage updating with the name of this service.

Reply:
Done